en/getting_started/px4_basic_concepts.md

```diff
@@ -128,13 +128,16 @@ The companion computer will usually communicate using a MAVLink API like the MAV
 * [Robotics APIs](https://dev.px4.io/master/en/robotics/) (PX4 Developer Guide)
 
 
-## Removable Memory/Logging
+## SD Cards (Removable Memory) {#sd_cards}
 
-PX4 uses SD memory cards for storing [flight logs](../getting_started/flight_reporting.md) (SD support may not be present on every flight controller).
+PX4 uses SD memory cards for storing [flight logs](../getting_started/flight_reporting.md), and they are also required in order to use UAVCAN peripherals and fly [missions](../flying/missions.md).
+
+By default, if no SD card is present PX4 will beep twice during boot (and none of the above features will be available).
 
 > **Tip** The maximum supported SD card size on Pixhawk boards is 32GB.
+  The *SanDisk Extreme U3 32GB* is [highly recommended](https://dev.px4.io/master/en/log/logging.html#sd-cards) (Developer Guide).
 
-A number of recommended cards are listed in: [Developer Guide > Logging](http://dev.px4.io/en/log/logging.html#sd-cards)
+SD cards are never-the-less optional, and some flight controllers may not even include an SD Card slot (in this case the notification beeps are disabled using the parameter [CBRK_BUZZER](../advanced_config/parameter_reference.md#CBRK_BUZZER)).
```

Comment:
The CBRK_BUZZER is too low level for this topic, but there isn't a better place AFAIK for putting this.

Reply:
Not sure if you want to mention this here as well:
- logs can be streamed to another compoment (companion)
- missions can be kept in RAM+FLASH

Reply:
@bkueng  Re ^^^

I'd love to, but it isn't clear how you would do this - in particular the missions can be kept in RAM+Flash. Can you tell me how?

Reply:
Discussion continued in https://github.com/PX4/px4_user_guide/pull/696/files#r398280705